frontends/mit-open/webpack.config.js

```diff
@@ -238,7 +236,7 @@ module.exports = (env, argv) => {
           : [new ReactRefreshWebpackPlugin()],
       )
       .concat(
-        ENVIRONMENT !== "local" && WEBPACK_ANALYZE === "True"
+        WEBPACK_ANALYZE === "True"
```

Comment:
This will be a boolean once handled by envalid, never string "True". I think envalid will throw if set as anything other that `"true"` or `"false"`.